chalice/utils.py

```diff
@@ -101,6 +116,84 @@ def set_file_contents(self, filename, contents, binary=True):
         with open(filename, mode) as f:
             f.write(contents)
 
+    def unpack_zipfile(self, zipfile_path, unpack_dir):
+        # type: (str, str) -> None
+        with zipfile.ZipFile(zipfile_path, 'r') as z:
+            z.extractall(unpack_dir)
+
+    @contextlib.contextmanager
+    def zipfile_context(self, filename, mode='r',
+                        compression=zipfile.ZIP_STORED):
+        # type: (str, str, int) -> Iterator[zipfile.ZipFile]
+        try:
+            z = zipfile.ZipFile(filename, mode=mode, compression=compression)
+            yield z
+        finally:
+            z.close()
+
+    def unpack_tarfile(self, tarfile_path, unpack_dir):
+        # type: (str, str) -> None
+        with tarfile.open(tarfile_path, 'r:gz') as tar:
+            tar.extractall(unpack_dir)
+
+    def directory_exists(self, path):
```

Comment:
These should probably stay close to the names of the functions they're aliasing.